ui/src/logs/components/LogViewerChart.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {PureComponent} from 'react'
+import LineGraph from 'src/shared/components/LineGraph'
+import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
+
+import {TimeRange} from 'src/types'
+
+interface Props {
+  onZoom: (lower: string, upper: string) => void
+  timeRange: TimeRange
+  data: object[]
+}
+
+class LogViewerChart extends PureComponent<Props> {
+  public render() {
+    const {timeRange, data, onZoom} = this.props
+    return (
+      <LineGraph
+        onZoom={onZoom}
+        queries={[]}
```

Comment:
is this something that could be set as a defaultProp in `LineGraph`? strange to see a fixed `[]` as the value for `queries`

Reply:
this is just a place holder right now.

Reply:
:thu